### ISSUE TITLE
this repo is not private

### ISSUE THREAD

**Issue by Mustwey** [liusida/ComfyUI-dockers#1]

**Comment by liusida** [comment]
Yeah, thanks for reminding me. I've made this repo public. Did I accidentally leave my key or password somewhere?

**Comment by Mustwey** [comment]
i didn't bother checking, yet u can search keywords to find if u did store your passes somewhere
